### DISCUSSION TITLE
Comparison of Zigbee thermostatic heads - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#50] [Comments]
# Comparison of Zigbee thermostatic heads - Haade.fr

Showing of 3 Zigbee thermostatic heads Xiaomi Aqara SRTS-A01, Sonoff TRVZB and Moes ZTRV-BY-100, a comparison to find out which is the best faucet in the contest

https://haade.fr/en/blog/comparative-thermostat-head-tap-zigbee-xiaomi-sonoff-moes

**Comment by helgek** [comment]
Thank you for this review. I’ve been using the Aqara trvs now for about a year and I’m overall satisfied. I chose them because of external temperature configuration.
